services/movie_session.py

```diff
@@ -26,11 +27,11 @@ def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
 
 def update_movie_session(
     session_id: int,
-    show_time: str = None,
-    movie_id: int = None,
-    cinema_hall_id: int = None,
+    show_time: Optional[str] = None,
+    movie_id: Optional[int] = None,
+    cinema_hall_id: Optional[int] = None,
 ) -> None:
-    movie_session = MovieSession.objects.get(id=session_id)
+    movie_session: MovieSession = MovieSession.objects.get(id=session_id)
```

Comment:
The `get` method will raise a `DoesNotExist` exception if no movie session with the given ID is found. Consider handling this exception to provide a more user-friendly error message or behavior.